src/ts/components/core/radio/Radio.tsx

```diff
@@ -36,6 +36,8 @@ interface Props
     /** Determines whether icon color with filled variant should depend on `background-color`. If luminosity of the `color` prop is less than `theme.luminosityThreshold`, then `theme.white` will be used for text color, otherwise `theme.black`. Overrides `theme.autoContrast`. */
     autoContrast?: boolean;
     /** To be used with Radio group */
+    disabled?: boolean;
+    /** Determines whether Radio disabled and non-selectable */
```

Comment:
The docstring needs to be above the prop.  It should look like:
```
/** Determines whether Radio disabled and non-selectable */
disabled?: boolean;
```

Also, please move it so that it works correctly with the other props.  It currently is located between the `value` prop and the `value` prop's docstring.